api/features/permissions.py

```diff
@@ -124,16 +125,27 @@ def has_permission(self, request: Request, view: GenericViewSet) -> bool:
     def has_object_permission(
         self, request: Request, view: GenericViewSet, obj: FeatureState
     ) -> bool:
-        permission = (
-            MANAGE_SEGMENT_OVERRIDES if obj.feature_segment_id else UPDATE_FEATURE_STATE
-        )
+        if isinstance(obj, FeatureExternalResource):
```

Comment:
I think we should extend this permission class instead of modifying it?

Reply:
I created a new permission for the featureExternalREsourceView inheriting from the FeaturePermission.

Reply:
Right. After talking with Matt to understand better the permissions schema, I have changed the base permission to `FeaturePermissions`, and extended it as `FeatureExternalResourcePermissions`